picard/extension_points/__init__.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
```

Comment:
What about putting this into a separate file `extension_points/item_actions.py`? I fear if we start putting this into `__init__.py` then we will end up with all different extension points being shoved into this file.

Reply:
Done in https://github.com/metabrainz/picard/pull/2454/commits/e11d2a6e68fd5233032137b054d85969bcd5f12b (and rebased)